pa/private.go

```
@@ -0,0 +1,410 @@
+						X:   ast.NewIdent("pa"),
+						Sel: ast.NewIdent("ZZZRequire"),
+					}
+				default:
+					return true
+				}
+
+				found = true
+
+				c.Replace(&ast.BlockStmt{
+					List: []ast.Stmt{
+						&ast.AssignStmt{ // zzz := pa.ZZZNew("2+2 == 5")
```

golangci-lint

commentedOutCode: may want to remove commented-out code

(gocritic)